packages/features/shell/Shell.tsx

```diff
@@ -898,11 +900,6 @@ function SideBar({ bannersHeight, user }: SideBarProps) {
                   isLocaleReady ? "hover:bg-emphasis hover:text-emphasis" : "",
                   index === 0 && "mt-3"
                 )}
-                aria-current={
-                  defaultIsCurrent && defaultIsCurrent({ item: { href: item.href }, router })
-                    ? "page"
-                    : undefined
```

Comment:
Can't be current with the options 

view_public_page -> opens in a new tab
copy_public_page_link -> copies and isnt a page
settings -> layout changes and so no point doing this calc